src/app/ngrx/effects/cms.effects.ts

```diff
@@ -94,4 +129,48 @@ export class CmsEffects {
     params['episodes'] = episodeIds.join(',');
     this.router.navigate([this.routeParams['seriesId'], 'downloads', this.routeParams['interval'], params]);
   }
+
+  private docToPodcast(doc: HalDoc): Observable<PodcastModel> {
```

Comment:
Looks like we can actually fully get rid of that `doc: HalDoc` property on the models now. I had originally put that in there because I was querying stories off the series doc, and I immediately hated it.